src/data/markdown/docs/30 xk6-browser/01 xk6-browser/01 Browser/newcontext--options--.md

```diff
@@ -21,35 +21,35 @@ You can customize the creation of a new [BrowserContext](/javascript-api/xk6-bro
-| reducedMotion                       | string  | Minimizes the amount of motion by emulating the 'prefers-reduced-motion' media feature. It can be one of `'reduce'` and `'no-preference'`. See [page.emulateMedia()](/javascript-api/xk6-browser/page#page-emulatemedia-options) for the options. Defaults to `'no-preference'`. |
-| screen                              | object  | Sets a window screen size for all pages in the context. It can only be used when the viewport is set. See: [screen](#screen-options) for the options.                                                                                                                            |
-| timezoneID                          | string  | Changes the context's timezone. See [ICU's metaZones.txt](https://cs.chromium.org/chromium/src/third_party/icu/source/data/misc/metaZones.txt?rcl=faee8bc70570192d82d2978a71e2a615788597d1) for a list of supported timezone IDs.                                                |
-| userAgent                           | string  | Specifies the user agent to use in the context.                                                                                                                                                                                                                                  |
-| viewport                            | object  | Sets a viewport size for all pages in the context. `null` disables the default viewport. See: [viewport](#viewport-options) for the options. Defaults to `1280x720`.                                                                                                             |
+| Option                              | Type    | Default         | Description                                                                                                                                                                                                                                       |
+|-------------------------------------|---------|-----------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| bypassCSP                           | boolean | `false`         | Whether to bypass a page's Content-Security-Policy.                                                                                                                                                                                               |
+| colorScheme                         | string  | `light`         | Whether to display a page in dark or light mode by emulating the 'prefers-colors-scheme' media feature. It can be one of `'light'`, `'dark'`, `'no-preference'`.                                                                                  |
```

Comment:
```suggestion
| colorScheme                         | string  | `'light'`         | Whether to display a page in dark or light mode by emulating the 'prefers-colors-scheme' media feature. It can be one of `'light'`, `'dark'`, `'no-preference'`.                                                                                  |
```

Same for other options where `colorScheme` is used, and for any other string values.